snapcraft/internal/sources/_subversion.py

```diff
@@ -81,8 +81,11 @@ def _get_source_details(self):
         commit = self.source_commit
 
         if not commit:  # retrieve the commit id
-            lines = subprocess.check_output(['svn', 'info', self.source_dir]
-                                            ).decode('utf-8').split('\n')
+            env = dict(os.environ)
```

Comment:
Please use `os.environ.copy()` here instead.

Reply:
Sure, I'll do

Reply:
Oh sorry, I haven't seen your request about the test. I'll add the test in the next commit